src/Services/UserService.ts

```diff
@@ -67,4 +66,23 @@ export class UserService {
         }
         return options
     }
+
+    public async updateAccount(userId: number, email: string, password: string) {
```

Comment:
Maybe it's better to pass the whole object and sequelize will handle updates that are there. If we add more properties, you'll have many else/ifs